index.html

```diff
@@ -542,6 +542,49 @@ <h2 id="constrainable-properties">
             </tr>
           </tbody>
         </table>
+        <p>The following new and existing <code><a>MediaStreamTrack</a></code>
+        <a href="https://www.w3.org/TR/mediacapture-streams/#constrainable-properties">
+        Constrainable Properties</a> are defined to apply to the user-selected
+        audio sources, with the following behavior:</p>
+        <table class="simple">
+          <thead>
+            <tr>
+              <th>Property Name</th>
+              <th>Type</th>
+              <th>Behavior</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr id="def-constraint-width">
+              <td><dfn>originAudio</dfn></td>
+              <td><code><a href="https://www.w3.org/TR/mediacapture-streams#dom-constrainboolean">ConstrainBoolean</a></code></td>
+              <td>
+                <p>As a setting, a value of <code>true</code> indicates that
+                audio produced by this origin is included in the captured audio,
+                a value of <code>false</code> indicates that audio from the
+                origin is not included in the captured audio.</p>
+                <p>As a constraint, this property can be constrained to the
+                value <code>false</code>, producing a source incapable of
+                including origin audio (the capability of such a source is the
+                lone value <code>false</code>). However, attempts to constrain
+                this property to the value <code>true</code> MUST result in
+                <code>OverconstrainedError</code>. That is, the application can
+                force the exclusion of the origin audio but it cannot force the
+                inclusion any particular audio source. If this property is
+                unconstrained, the resulting source MAY be capable of either the
+                value <code>true</code> or <code>false</code>.</p>
+                <div class="note">
+                  <p>Due to inherrent difficulties in isolating different audio
+                  sources on a system, the recommented solution to fulfilling
+                  <code>originAudio:false</code> is to exclude user choices. For
+                  example, a user agent that supports tab capture could fulfill
+                  this constraint by presenting the user with choices including
+                  all tabs except for the origin tab.</p>
```

Comment:
Or we could define it as best-effort. Sites could check `track.getSettings().ownAudioCancellation == false` to see if cancellation is happening or whether they need to significantly lower the volume of participants.

Reply:
Good points. `restrictOwnAudio` might be another name then, if we want to imply something stronger than cancellation at times. The important point for me was that it's within the realm of a processing constraint. How UAs solve it I guess is up to them. As to limiting user choice, at least we're limiting *away* from "scary" sources in this way, so there may be some leeway here.

Reply:
I don't see how removing the option to share audio on choices that would result in a muted track would  be a fingerprint/privacy/security issue; there is no way for the application to know whether the user de-selected audio or if the user agent did it for them. Furthermore, since audio is optional, excluding audio does not impact whether or not the getDisplayMedia() request would succeed or fail.

Reply:
[Not all sources are equal from a security perspective](https://blog.mozilla.org/webrtc/share-browser-windows-entire-screen-sites-trust/). The reason to limit user choice was not based on fingerprinting risk, rather the risk of letting web sites steer users toward sharing the riskier class of sources, specifically *web surfaces*, and an *entire desktop* on which a web surface is present.

Here we're ok since we're generally talking about steering people *away* from those sources. Doubly-so if we're only talking about muting audiovisual sources instead of removing them from selection entirely.